src/iplant_groups/routes/folders.clj

```diff
@@ -8,69 +8,80 @@
 
 (defroutes folders
   (GET "/" []
-        :query       [params SearchParams]
-        :return      FolderList
-        :summary     "Folder Search"
-        :description "This endpoint allows callers to search for folders by name. Only folders
-        that are visible to the given user will be listed."
-        (ok (folders/folder-search params)))
```

Comment:
This was a minor change, but I think the reduced indentation helps with nested, contexts.